### ISSUE TITLE
Untrack embed-test.log and related generated files

### ISSUE THREAD

**Issue by yb66** [ffi/ffi#673]
Hi,

It appears to me that #544b7210 introduced some extra files to help with running the tests. Either they're generated and can be ignored, or they should be checked in. I'm assuming the former unless told otherwise.

Regards,
iain

**Comment by larskanis** [comment]
You're right. Could you please send a pull request, since you already fix this?

**Comment by yb66** [comment]
Hi @larskanis,

My mistake, I thought referencing it that way would automatically open the request. I've opened the request, #674.

Regards,
iain
